src/Serilog.Sinks.MongoDB/project.json

```diff
@@ -0,0 +1,21 @@
+﻿{
+  "version": "2.0.0-beta-*",
+  "description": "The MongoDB sink for Serilog",
+  "authors": [ "Kiran Makkapati", "Serilog Contributors" ],
+  "tags": [ "serilog", "mongodb" ],
+  "projectUrl": "http://serilog.net",
+  "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+  "iconUrl": "http://serilog.net/images/serilog-sink-nuget.png",
+  "dependencies": {
+    "Serilog": "2.0.0-beta-511",
+    "Serilog.Sinks.PeriodicBatching": "2.0.0-beta-700"
+  },
+
+  "frameworks": {
+    "dnx46": {
```

Comment:
Yeah, I guess we have to go through a checklist and change all projects before shipping, but it would be nice if things were somewhat aligned with the core project before we started the RC2 transition. In theory, it shouldn't be much more than just search/replacing `dotnet5.x` with `netstandard1.y` and updating some dependency versions. Of course, there's also the tooling change, which affects our build script(s), but that could hopefully be shared (copied?) across all projects.